.github/workflows/ci_nightly.yml

```diff
@@ -36,6 +36,20 @@ jobs:
       - uses: taiki-e/install-action@9b5b983efc779f85e5e5d11539f005e85ccb27ff
         with:
           tool: just,cargo-nextest
-
       # Run the tests:
-      - run: just nextest-slow ${{ matrix.type }}
+      - uses: mathiasvr/command-output@v2.0.0
```

Comment:
Lets always pin a specific commit :) Much harder to find a hash collision than to force push a tag.

Reply:
done